File: SQLitecodeforTechTitanstoregistervariousroles/app/src/main/java/com/example/sqlitecodefortechtitanstoregistervariousroles/MainActivity.java
package com.example.sqlitecodefortechtitanstoregistervariousroles;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {
    Button btnAdd, btnUpdate, btnDelete, btnViewAll;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        btnAdd = findViewById(R.id.buttonAdd);
        btnUpdate = findViewById(R.id.buttonUpdate);
        btnDelete = findViewById(R.id.buttonDelete);
        btnViewAll = findViewById(R.id.buttonViewAll);
        btnAdd.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, AddDataActivity.class);
            startActivity(intent);
        });
        btnUpdate.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, UpdateDataActivity.class);
            startActivity(intent);
        });
        btnDelete.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, DeleteDataActivity.class);
            startActivity(intent);
        });
        btnViewAll.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, ViewAllDataActivity.class);
            startActivity(intent);
        });
    }
}

File: DevConnects/app/src/main/java/com/example/devconnects/MainActivity.kt
package com.example.devconnects

import android.content.Intent
import android.graphics.Color
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.google.firebase.Firebase
import com.google.firebase.FirebaseApp
import com.google.firebase.auth.FirebaseAuth

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        window.statusBarColor = Color.TRANSPARENT

        Handler(Looper.getMainLooper()).postDelayed({

            if (FirebaseAuth.getInstance().currentUser == null)
                startActivity(Intent(this, SignUpActivity::class.java))
            else
                startActivity(Intent(this, HomeActivity::class.java))
            finish()
        }, 1000) // 3000 is the delayed time in milliseconds.
    }
}